dht/cdc_partitioner.hh

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2020 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "i_partitioner.hh"
+
+namespace dht {
+
+class cdc_partitioner final : public i_partitioner {
+public:
+    cdc_partitioner() = default;
+    virtual const sstring name() const { return "com.scylladb.dht.CDCPartitioner"; }
```

Comment:
This is a duplicating of the string in "registrator". Can we have name() call registrator, or the other way around, or put this string once in a string constant?

Reply:
Will give it a try.

Reply:
done.